charts/merlin/README.md

```diff
@@ -339,6 +339,7 @@ The following table lists the configurable parameters of the Merlin chart and th
 | mlp.fullnameOverride | string | `"mlp"` |  |
 | mlp.keto.enabled | bool | `true` |  |
 | mlp.keto.fullnameOverride | string | `"mlp-keto"` |  |
+| rendered.overwrites | object | `{}` |  |
```

Comment:
what if we name it `rendered.overrides`?

Reply:
changed